01week/rockPaperScissors.js

```diff
@@ -7,12 +7,51 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
-
+// Ah Rock Paper scissors
+// Ties: rock:rock; paper:paper; scissors:scissors
+// Hand1 wins: rock:scissors; paper:rock; scissors:paper
+// Hand2 wins: scissors:rock; rock:paper; paper:scissors
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  hand1 = hand1.toLowerCase().trim();
+  hand2 = hand2.toLowerCase().trim();
 
-}
+  let winningMessage = '';
+
+  if ( (hand1 !== 'rock' && hand1 !== 'paper' && hand1 !== 'scissors') ||
+       (hand2 !== 'rock' && hand2 !== 'paper' && hand2 !== 'scissors')
+     ) {
+    winningMessage = "Alright, someone did not enter a valid value";
+    return winningMessage;
+  }
+
+  if (hand1 === hand2) {
+    winningMessage = "It's a tie!"
+  }
+  else if (hand1 === 'rock') {
+    if (hand2 === 'paper') {
+      winningMessage = "Hand two wins!";
+    } else {
+      winningMessage = "Hand one wins!";
+    }
+  } // else if hand 1 = rock
+  else if (hand1 === 'paper') {
+    if (hand2 === 'rock') {
+      winningMessage = "Hand one wins!";
+    } else {
+      winningMessage = "Hand two wins!";
+    }
+  } // else if hand 1 = paper
```

Comment:
Piero, just a note that I have updated my comments in this code to read English.  Hoping to raise my  grade.  See below commit.